azure-spring-boot-samples/azure-active-directory-spring-boot-backend-sample/README.md

```diff
@@ -0,0 +1,68 @@
+### How to configure
+
+#### Register your application with your Azure Active Directory Tenant
+
+Follow the guide [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code#register-your-application-with-your-ad-tenant).
+
+#### Configure groups for sign in user
+
+In order to try the authorization action with this sample with minimum effort, [configure the user and groups in Azure Active Directory](https://docs.microsoft.com/en-us/azure/active-directory/active-directory-groups-create-azure-portal), configure the user with `group1`. 
+
+
+#### Configure application.properties
+
+```properties
+spring.security.oauth2.client.registration.azure.client-id=xxxxxx-your-client-id-xxxxxx
+spring.security.oauth2.client.registration.azure.client-secret=xxxxxx-your-client-secret-xxxxxx
+# The client registration provider must be azure to take advantage of the auto-configuration
+spring.security.oauth2.client.registration.azure.provider=azure
```

Comment:
Shall we make it default to azure? any other value than 'azure' possible?

Reply:
Thanks for the review, it's better, already fixed as you may know..